WeakAurasOptions/WeakAurasOptions.toc

```diff
@@ -38,6 +38,10 @@ SubRegionOptions\Border.lua
 SubRegionOptions\Glow.lua
 SubRegionOptions\Tick.lua
 SubRegionOptions\Model.lua
+SubRegionOptions\StopMotion.lua
+SubRegionOptions\Texture.lua
+SubRegionOptions\CircularProgressTexture.lua
+SubRegionOptions\LinearProgressTexture.lua
```

Comment:
This needs to be added to WeakAurasOptions_Vanilla.toc and WeakAurasOptions_Cata.toc